presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environment.java

```diff
@@ -58,6 +58,10 @@ public Environment(String name, Map<String, DockerContainer> containers)
     public void start()
     {
         try {
+            containers.entrySet().stream()
+                    .filter(e -> !e.getKey().equals("tests"))
+                    .map(Map.Entry::getValue)
+                    .forEach(c -> c.withReuse(true));
```

Comment:
Evnrionment should not be concerned about specific containers in it.
If we want to apply some behavior to all containers, but not the test container,
we can do so:

- apply default in `io.prestosql.tests.product.launcher.env.DockerContainer`
- apply override in `io.prestosql.tests.product.launcher.env.common.Standard#createTestsContainer`